lib/dams/controller_helper.rb

```diff
@@ -675,8 +675,12 @@ def audit( id = "unknown" )
     def referrer_controller( request )
       uri = URI(request.referrer || "")
       if uri.host == request.host
```

Comment:
It looks like Rails.root is the path to the rails app but not the application root, and we can think about using RAILS_RELATIVE_URL_ROOT but I am seeing it ambiguous in test environment with a root like /. Maybe we can think about fixing the failed tests, or testing those failed tests in other ways?